invenio_orcid/utils.py

```diff
@@ -31,3 +31,30 @@ def get_authors_credentials(orcid):
         id=orcid, method='orcid').first()
     user = RemoteAccount.query.filter_by(user_id=raw_user.id_user).first()
     return user.tokens[0].access_token
+
+
+def convert_to_orcid(record):
```

Comment:
I think it's project-specific. 
The goal  of this is givena the record of a paper to produce a json document that orcid will accept.
So that means that there is no generic implementation (as fas as I can think).